test/adapters/file/write.spec.js

```diff
@@ -102,7 +102,8 @@ describe('write file adapter tests', function () {
                 return run_read_file_juttle(tmp_file, { format: format });
             })
             .then(function(result) {
-                expect(result.errors).deep.equals([]);
+                expect(result.errors.length).to.equal(0);
```

Comment:
`expect(result.errors).deep.equals([]);` is actually superior because if it fails it fails with the message `Expected ["some actual error message"] to deep equal []` as opposed to the useless `Expected 1 to equal 0`.